server/src/main/java/org/opensearch/extensions/rest/RestSendToExtensionAction.java

```diff
@@ -45,6 +48,13 @@ public class RestSendToExtensionAction extends BaseRestHandler {
     private static final String SEND_TO_EXTENSION_ACTION = "send_to_extension_action";
     private static final Logger logger = LogManager.getLogger(RestSendToExtensionAction.class);
     private static final String CONSUMED_PARAMS_KEY = "extension.consumed.parameters";
+    // To replace with user identity see https://github.com/opensearch-project/OpenSearch/pull/4247
+    private static final Principal DEFAULT_PRINCIPAL = new Principal() {
+        @Override
+        public String getName() {
+            return "OpenSearchUser";
```

Comment:
This needs a test.

Reply:
This should be tested inside the test for `prepareRequest` method. Is there a place where this test is already written?